src/ScoutingMain.elm

```diff
@@ -18,7 +19,7 @@ main : Program () Model Msg
 main =
     Browser.element
         { init = always ( init, Cmd.none )
-        , view = view >> layout []
+        , view = view >> layout [ width fill ]
```

Comment:
Oh! So it was here!!
I've looked ages for the part that limited it despite all of the others having `width fill`!